api/test/configure.ts

```diff
@@ -0,0 +1,743 @@
+/**
+ * Create an environment with users and super admins
+ */
+import { AuthenticationResult } from '@feathersjs/authentication/lib'
+import { LocalStrategy } from '@feathersjs/authentication-local/lib/strategy'
+// import { Paginated } from '@feathersjs/feathers'
+
+import { USER_PROFILE } from '@locokit/definitions'
+
+import { createApp } from '../src/app'
+// import { Workspace } from '../models/workspace.model'
+
+// import { LckAclSet } from '../models/aclset.model'
+// import { Group } from '../models/group.model'
+import { UserResult } from '../src/services/auth/user/user.schema'
+// import { Database } from '../models/database.model'
+// import { Table } from '../models/table.model'
+// import { TableColumn } from '../models/tablecolumn.model'
+// import { TableRow } from '../models/tablerow.model'
+// import { dropWorkspace } from '../utils/dropWorkspace'
+
+const app = createApp()
+
+export interface SetupData {
+  // workspace1Id: string
+  // workspace2Id: string
```

Comment:
Maybe we can add a schema or few sentence to explain data:
By example: User1 is a membre from workspace1…

Reply:
What do you hear by "schema" ?
A comment maybe ?

This setup is incomplete for now.
As no workspace is created, only users.

I think this file will be upgraded in the next PR for testing group / role / workspace.
As it could evolve between now and the next PR ?

Reply:
Schema = draw

In the previous version, it was difficult to track who owns a workspace and which users/groups are in that workspace, which is why I suggest to make a diagram/schema/sentences…

It is not needed for this specific mr.

Reply:
Ok ! I understand better.
That's a really good idea indeed.

Maybe this could go in a documentation dedicated to the developers ?
Explaining that there is a setup available when making tests.
We could add an illustration in this page to explain how we create this setup ?

What do you think ?

Reply:
Good idea. Can you create a ticket related to this discussion?

Reply:
https://github.com/locokit/locokit/issues/210